projects/lights_control/inc/lights_config.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include "gpio.h"
+
+typedef enum {
+  LIGHTS_CONFIG_BOARD_TYPE_FRONT = 0,
+  LIGHTS_CONFIG_BOARD_TYPE_REAR,
+  NUM_LIGHTS_CONFIG_BOARDS
+} LightsConfigBoardType;
+
+typedef struct LightsConfig {
+  const LightsConfigBoardType board_type;
+  const GPIOAddress *addresses;
+  const uint8_t num_addresses;
+  const GPIOSettings *gpio_settings_out;
+  const uint16_t (*event_mappings)[2];
+  const uint8_t num_supported_events;
+} LightsConfig;
+
+typedef enum {
+  LIGHTS_CONFIG_FRONT_LIGHT_HORN = 0,
+  LIGHTS_CONFIG_FRONT_LIGHT_HIGH_BEAMS_RIGHT,
+  LIGHTS_CONFIG_FRONT_LIGHT_HIGH_BEAMS_LEFT,
+  LIGHTS_CONFIG_FRONT_LIGHT_LOW_BEAMS_RIGHT,
+  LIGHTS_CONFIG_FRONT_LIGHT_LOW_BEAMS_LEFT,
+  LIGHTS_CONFIG_FRONT_LIGHT_DRL_RIGHT,
+  LIGHTS_CONFIG_FRONT_LIGHT_DRL_LEFT,
+  LIGHTS_CONFIG_FRONT_LIGHT_SIDE_LEFT_INDICATOR,
+  LIGHTS_CONFIG_FRONT_LIGHT_LEFT_TURN,
+  LIGHTS_CONFIG_FRONT_LIGHT_SIDE_RIGHT_INDICATOR,
+  LIGHTS_CONFIG_FRONT_LIGHT_RIGHT_TURN,
+  NUM_FRONT_LIGHTS
+} LightsConfigFrontLights;
```

Comment:
If you move left/right config into their own files you should move these too. They shouldn't be used anywhere else anyways.